ApiUsage/ArmBased/README.md

```diff
@@ -37,9 +37,8 @@ Instructions:
 3. Go to Program.cs and populate `AccountName` with your account name
 4. Go to Program.cs and populate `VideoUrl` with your video url
 5. make sure dotnet 6.0 is installed. if not, please install https://dotnet.microsoft.com/download/dotnet/6.0
-6. login to azure `az login --use-device`
-7. Open your terminal and navigate to VideoIndexerArm folder
-8. Run dotnet build
+6. Open your terminal and navigate to "media-services-video-indexer\ApiUsage\ArmBased" folder
```

Comment:
media-services-video-indexer is nit a folder it the repo name. 
navigate to ApiUsage since you are already have cloned this repo